engine/crates/engine/src/registry/cache_control.rs

```diff
@@ -8,129 +8,129 @@ pub enum CacheControlError {
     Validate(Vec<crate::ServerError>),
 }
 
-#[cfg(todo)]
-#[cfg(test)]
-mod tests {
-    use engine_parser::types::OperationType::Query;
+// #[cfg(todo)]
```

Comment:
Could probably just delete this tbh, I think it's mostly covered by integration tests